crates/pallet-domains/src/staking.rs

```diff
@@ -2387,169 +2399,173 @@ pub(crate) mod tests {
     /// Their balance should be almost the same before and after.
     #[test]
     fn prop_withdraw_excess_operator_stake_with_nominator() {
-        prop_test!(&(OPERATOR_STAKE_RANGE, NOMINATOR_STAKE_RANGE,), |(
```

Comment:
From this point onward in this file, the renames caused formatting and indentation changes. But nothing else has changed.